### ISSUE TITLE
Exceptions

### ISSUE THREAD

**Issue by carlcs** [spatie/pdf-to-image#3]
How's the throw exceptions meant to work? I'm using the package with composer autoloading and the exceptions classes do not get loaded (other namespace). Am I meant to provide my own?

**Comment by freekmurze** [comment]
I'm not sure I understand your problem correctly.

You can catch the exceptions from this package using regular php

```php
...
catch(\Spatie\ConvertPdfToImage\Exceptions $exception) {
....
```

**Comment by carlcs** [comment]
Ok, but what's the src/Exceptions/ folder for when autoloader doesn't load the classes because of the different namespaces Spatie\ConvertPdfToImage and Spatie\PdfToImage.

**Comment by freekmurze** [comment]
The exceptions were not namespaced correctly. I fixed this in the newly tagged v1.0.3. and added some test to make sure they work properly now.

Thanks for bringing this to my attention.
